code/game/gamemodes/blob/overmind.dm

```diff
@@ -87,13 +87,15 @@
 	if (!message)
 		return
 
-	var/message_a = say_quote("\"[html_encode(message)]\"")
+	var/message_a = say_quote("\"[message]\"")
```

Comment:
It was being double-encoded.
sanitize() is called earlier which ALSO calls html_encode.
So it was only de-coded once which ended up in #234234; whatever HTML entity stuff showing up instead of just '